src/ipc/ipc3/helper.c

```diff
@@ -285,8 +285,10 @@ static void comp_specific_builder(struct sof_ipc_comp *comp,
 #endif
 		break;
 	default:
+		return -EINVAL;
 		break;
```

Comment:
please, remove this `break`

Reply:
Argh, my sense for coding style may have been permanently damaged due to MISRA-C work I did it my past life.